feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -70,35 +70,47 @@ var (
 		Desc:    "dutPort1",
 		IPv4:    "192.0.2.1",
 		IPv4Len: 30,
+		IPv6:    "2001:db8::192:0:2:1",
+		IPv6Len: 126,
```

Comment:
Should all these be updated to match the ate equivalent?
https://github.com/openconfig/featureprofiles/blob/4b62cb22e7cff45cbb8f697af3f94a9232e6d6ce/feature/gribi/ate_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go#L66-L96

Reply:
Not sure why IPv6 interfaces are needed here as the traffic is IPv4 only

Reply:
Similar to changes in ate_tests. Refer: https://github.com/openconfig/featureprofiles/pull/859

Reply:
> Not sure why IPv6 interfaces are needed here as the traffic is IPv4 only

Removed ipv6 interfaces